### PR TITLE
add June 7 meeting notes

### PR DESCRIPTION
**Pull request by dcwalk**

### CODE REVIEW

File: meeting_notes/20160607_meeting-notes.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+- [Project Under Gardiner website](http://www.undergardiner.com/)
+- Jon at the walkshop for the Under Gardiner, they indicated they are speaking with Beanfield to provide free wifi hotspot
+    - Outreach lead is interested in connecting with us and seeing what could be related
+- Do we fit into the 'local services' niche? Nodes that host 'local' content
+- Related? Scadding Court has a lack of wifi, Victoria Hub pilot project for wifi at a spot
+    - Revisit in July roadmap planning update
+    - Jon will continue to keep tabs on these projects
+
+## Outreach Update
```
Comment (benhylau) [inline meeting_notes/20160607_meeting-notes.md:54]: cc. @Famicoman

File: meeting_notes/20160607_meeting-notes.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+- tomesh.101 folder structure, **VOTED** and consensus to adopt restructured repo:
```
Comment (benhylau) [inline meeting_notes/20160607_meeting-notes.md:56]: @dcwalk Should this restructure become a Wekan card?

File: meeting_notes/20160607_meeting-notes.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+    - [Introduction to networks event posting](http://makerfestival.ca/events/introduction-to-networks-from-the-internet-to-tomesh-net/)
+    - Registration: **hello@tomesh.net**, dcwalk will manage
+    - Attendance: dcwalk, ben, udit
+    - Planning Meeting on **Mon Jun 20**​, Run-through on ​**Fri Jun 24**
+
+## Infrastructure Update: Wekan  
+
+- Board is up [wekan.tomesh.net](http://wekan.tomesh.net)
+    - Current status: demoing, ssl needs to be deployed, rocket.chat
+    - Integrations: rocket.chat is close; unsure about GitHub
```
Comment (benhylau) [inline meeting_notes/20160607_meeting-notes.md:40]: Rocket-Wekan: https://discuss.wekan.io/t/rocket-chat-integration/15
Rocket-Github: https://github.com/RocketChat/Rocket.Chat/issues/2339
Wekan-Github: integration: https://github.com/wekan/wekan/issues/253

File: meeting_notes/20160607_meeting-notes.md (hunk 2 above)
Comment (dcwalk) [inline meeting_notes/20160607_meeting-notes.md:56]: 💯 here: http://wekan.tomesh.net/b/LWS8X7sGFXqDgZ7ag/tomesh-net/smpZ3zNJSp48mxnSt

**Pull request merged**